app/controllers/users_controller.rb

```diff
@@ -55,7 +55,8 @@ def print_swipecard_with_pmb(swipecard, printer) # rubocop:todo Metrics/MethodLe
         printer,
         LabelPrinter::Label::Swipecard,
```

Comment:
Do you think it would be appropriate to store the label template name inside this `LabelPrinter::Label::Swipecard` class?

Reply:
It could be done but it would need need more thinking.
- Label class is providing values for placeholders only and it provides data. Template is doing the presentation.
- Template name is passed separately to PMB API as part of  (printer name, label data and template name)
- I checked Limber and a few other places, template and label data are handled separately.
- Passing template name to Label class would give a chance to make decisions based on template, but it does not look necessary.